tests/e2e/specs/swap-tokens.spec.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable ui-testing/no-disabled-tests */
+describe('Swap Tokens Tests', () => {
+  it(`should connect with Agoric Chain on https;//wallet.agoric.app`, () => {
+    cy.origin('https://wallet.agoric.app/', () => {
+      cy.visit('/');
+    });
+    cy.acceptAccess().then(taskCompleted => {
+      expect(taskCompleted).to.be.true;
+    });
+
+    cy.origin('https://wallet.agoric.app/', () => {
+      cy.visit('/wallet/');
+
+      cy.get('input.PrivateSwitchBase-input').click();
+      cy.contains('Proceed').click();
+
+      cy.get('button[aria-label="Settings"]').click();
+
+      cy.get('#demo-simple-select').click();
+      cy.get('li[data-value="local"]').click();
+      cy.contains('button', 'Connect').click();
+    });
+
+    cy.acceptAccess().then(taskCompleted => {
+      expect(taskCompleted).to.be.true;
+    });
+  });
+
+  it('should connect with wallet', () => {
+    cy.visit('/');
+
+    // Switch to local network
+    cy.get('button').contains('Agoric Mainnet').click();
+    cy.get('button').contains('Local Network').click();
+
+    // Click the connect button
+    cy.get('button').contains('Connect Keplr').click();
+    cy.get('input[type="checkbox"]').click();
+    cy.get('button:enabled').contains('Proceed').click();
+
+    // Accept access and confirm
+    cy.acceptAccess();
+    cy.get('button').contains('Keplr Connected').should('be.visible');
+  });
+
+  it('should swap tokens from IST to stable', () => {
+    let ISTbalance;
```

Comment:
our style is camel case
```suggestion
    let istBalance;
```